README.md

```diff
@@ -129,6 +131,55 @@ make build
 ```
 Don't forget to activate the app via Nextcloud web-gui.
 
+### Debugging
```

Comment:
@R0Wi, please insert an entry for this subsection into the table of contents.

Reply:
Done with https://github.com/R0Wi/workflow_ocr/pull/25/commits/936a1ca64ec59be4c1ab0f4012d527df284441b4

Reply:
Thanks for your commit to update the table of contents.